src/lib/components/ScriptureViewSofria.svelte

```diff
@@ -1062,6 +1062,5 @@ TODO:
         class:hidden={loading}
         style:font-size={fontSize}
         style:line-height={lineHeight}
-        class="single"
```

Comment:
This shouldn't be removed. It will later be taking the values from $viewMode, but I am not yet ready to do that. I need to finish the implementation of the BookCollectionSelector first.